Arithmetic/Division.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace CSharp_Practice.Arithmetic
+{
+    public class Division
+    {
+        public static void DivisionAndModule()
+        {
+            Console.WriteLine("Input two integers to get the division and rest of the division of the result");
+
+            int x = Convert.ToInt32(Console.ReadLine());
```

Comment:
Consider creating a custom ReadInt() function of your own that does validation and you can call it from all your code.